@types/pal/image.d.ts

```diff
@@ -0,0 +1,30 @@
+declare module 'pal/image' {
+    type ImageSource = import('pal/image/types').ImageSource;
+    type IMemoryImageSource = import('pal/image/types').IMemoryImageSource;
+    export class ImageData {
+        constructor (imageAsset?: ImageSource | ArrayBufferView);
+        //constructor (content: string, manifestRoot: string);
+        destroy (): void;
+        get src (): string;
+        set src(url: string);
+        set data(value: any);
+        get data(): ImageSource | ArrayBufferView;
+        set width(value: any);
+        get width(): number;
+        get height(): number;
+        set height(value: any);
+        get format(): number;
+        get compressed(): number;
+        get mipmapLevelDataSize(): number[] | undefined;
+        static downloadImage (
```

Comment:
`download` is a concept on web platforms, on many other platforms, we call it `load` instead.

could this be named as `loadImage` ?

Reply:
DownloadImg is implemented for every platform. It is downloading images from the server and DownloadImg for other platforms.

LoadImg can be implemented by defining an imagedata passed to null.